cloud.md

```diff
@@ -1,7 +1,15 @@
 # Cloud
 
+Once the virtual machines are instantiated in the cloud give some time for their kernel get updated. To re-use the same inventory file update the **/etc/hosts** with the respective IPs for each host.
```

Comment:
It is covered by:
https://github.com/nlesc-sherlock/emma/commit/58dcde4716b50dbd859973fbdb8b76f968c05f79